CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.4.1
+- add Stop and Start task (contribution by to Sebastian Schuberth [#79](https://github.com/novoda/gradle-android-command-plugin/pull/79))
```

Comment:
PR numbers are swapped here: PR #78 was the one adding the "Stop" task, PR #79 the one adding support for activity aliases.